app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailFragment.kt

```diff
@@ -75,6 +83,26 @@ class SessionDetailFragment : Fragment(), Injectable {
         }
 
         binding.toolbar.setNavigationOnClickListener { activity?.finish() }
+
+        binding.speakerSummary.viewTreeObserver.addOnPreDrawListener(
```

Comment:
Can you separate method like this https://github.com/DroidKaigi/conference-app-2018/blob/master/app/src/main/java/io/github/droidkaigi/confsched2018/presentation/speaker/SpeakerDetailFragment.kt#L109 ? 👀